USAGE.md

```diff
@@ -1,370 +1,707 @@
+The data in this repository is STIX2.0 and divided into three folders, one for each domain of ATT&CK. These domains generally follow the same format with a few departures. Domain differences will be noted in the relevant sections of this document. 
 
-### Migrating from MediaWiki
+STIX uses a different terminology than ATT&CK. The following table is a mapping of ATT&CK concepts to STIX 2.0 objects:
 
-### Common properties (on all objects)
-ATT&CK Property | ATT&CK MediaWiki | STIX Properties
---------------- | ---------------- | ---------------
-**Entry ID**    | `Has ID` | `external_references[i].external_id` where `external_references[i].source_name == "mitre-attack"`
-**Entry URL**   | `URL` | `external_references[i].url` where `external_references[i].source_name == "mitre-attack"`
-**Entry Title** | `Has display name` | `name`
-**Entry Text**  | `Has description` | `description`
-**Citation**    | `Citation reference` | `external_references`
-**Deprecated**  | `Deprecated` | `x_mitre_deprecated`
-**Revoked**     | `Not available via MediaWiki API` | `revoked`
-**Old ATT&CK ID** | `Not available via MediaWiki API` | `x_mitre_old_attack_id`
+| ATT&CK concept | STIX object type |
+|:------------|:----------|
+| [Matrix](#matrices)              | `x-mitre-matrix` |
+| [Tactic](#tactics)               | `x-mitre-tactic` |
+| [Technique](#techniques)         | `attack-pattern` |
+| [Sub-technique](#sub-techniques) | `attack-pattern` where `x_mitre_is_subtechnique = true` |
+| [Procedure](#procedures)         | `relationship` where `relationship_type = "uses"` and `target_ref` is an `attack-pattern` |
+| [Mitigation](#mitigations)       | `course-of-action` |
+| [Group](#groups)                 | `intrusion-set`  |
+| [Software](#software)            | `tool` or `malware` |
 
+Two additional object types are found in the ATT&CK catalog:
+ 
+| STIX object type | About |
+|:-----------------|:------|
+| `identity`       | Referenced in the `created_by_ref` of all objects to state that the MITRE Corporation created the object |
+| `marking-definition` | Referenced in the `object_marking_refs` of all objects to express the MITRE Corporation copyright |
+
+## Common departures from the STIX spec
+
+There are three general ways that ATT&CK departs from the stock STIX 2.0 format:
+- Custom object types. Object types prefixed with `x-mitre-`, e.g `x-mitre-matrix`, are custom STIX types not defined in the stock STIX spec. They follow the general [STIX Domain Object pattern](https://docs.oasis-open.org/cti/stix/v2.0/csprd01/part2-stix-objects/stix-v2.0-csprd01-part2-stix-objects.html#_Toc476230920) but describe concepts not covered by generic STIX2.0 types.
+
+- Extensions of existing object types. Fields not found in the stock STIX 2.0 spec are prefixed with `x_mitre_`, e.g `x_mitre_platforms` in `attack-patterns`. 
+    
+    All objects except relationships can have the following extended properties applied:
+
+    | field | type | description |
```

Comment:
This is nitpicking but there's another `Field-Type-Description` table a little further along where the headers are capitalized. It also looks a little weird these not being capitalized since the text within the description fields are capitalized.

Reply:
Good catch, I'll make sure to standardize capitalization.